chromium_src/chrome/common/url_constants.h

```diff
@@ -332,6 +332,10 @@ inline constexpr char16_t kPaymentMethodsURLForGPayWeb[] =
 inline constexpr char kAddressesAndPaymentMethodsLearnMoreURL[] =
     "https://support.brave.com";
 
+// The URL for the "Pay over time" support page.
+inline constexpr char16_t kPayOverTimeLearnMoreUrl[] =
```

Comment:
@bbondy should we really be doing this for every link? Anyone actually looking for this information is basically going to a dead end because we link them to support.brave.com for everything and we clearly don't have actual docs on this stuff. Not concerned about this one in particular, but this practice in general